rom-hacking/scripting/animation-commands/acmd-end.md

```diff
@@ -0,0 +1,29 @@
+# (777) _ACMD_END
+
+## Effect
+
+Ends the current animation.
+
+Resets _moveOffset to (0, 0, 0).
+
+Sets animation clip to 0 (Idle).
+
+Stops the stepping sound effect.
+
+Sets _forceDuration to -1.
```

Comment:
"Resets _moveOffset to (0, 0, 0)" and "Sets _forceDuration to -1" can be removed, I don't think they need to be mentioned.